ComponentKit/DataSources/CKCollectionViewTransactionalDataSource.mm

```diff
@@ -26,13 +26,14 @@ @interface CKCollectionViewTransactionalDataSource () <
 >
 {
   CKTransactionalComponentDataSource *_componentDataSource;
-  id<CKSupplementaryViewDataSource> _supplementaryViewDataSource;
+  __weak id<CKSupplementaryViewDataSource> _supplementaryViewDataSource;
   CKTransactionalComponentDataSourceState *_currentState;
   CKComponentDataSourceAttachController *_attachController;
 }
 @end
 
 @implementation CKCollectionViewTransactionalDataSource
+@synthesize supplementaryViewDataSource = _supplementaryViewDataSource;
```

Comment:
I *believe* this was necessary, since the ivar is explicitly defined....... But this was a while ago...